jupyterlab_caip_optimizer/src/components/sidebar_widget.tsx

```diff
@@ -33,6 +33,21 @@ import { RootState, AppDispatch } from '../store/store';
 import { Study } from '../types';
 import { setView } from '../store/view';
 import { Store } from 'redux';
+import { prettifyStudyName } from '../service/optimizer';
+import { style } from 'typestyle';
```

Comment:
Do you know if typestyle package was used in any other project?

Reply:
Yeah scheduler uses it. https://github.com/GoogleCloudPlatform/jupyter-extensions/blob/c1fb81d9c22575255dd372991aeda5df0321aed0/jupyterlab_gcpscheduler/src/components/action_bar.tsx#L30